static/templates/message_edit_form.hbs

```diff
@@ -7,7 +7,9 @@
     </div>
     <div class="edit-controls">
         {{> copy_message_button message_id=this.message_id}}
-        <textarea class="message_edit_content" maxlength="{{ max_message_length }}">{{content}}</textarea>
+        <div id="compose_banners"></div>
```

Comment:
We should not reuse an id. We may need another similar div here for edit state that behaves same as `#compose_banners`. Not too sure since I'm not very familiar with the working of the edit state in general

Reply:
I think that in order to use something in place of ```#compose_banners``` we will have to dupliacte ```compose_banner.ts```. I'll look further into this.